appengine/analytics/package.json

```diff
@@ -21,7 +21,8 @@
   },
   "dependencies": {
     "express": "4.16.2",
-    "got": "8.2.0"
+    "got": "8.2.0",
+    "semistandard": "^12.0.1"
```

Comment:
Shouldn't this be a dev dependency?  Here and below!

Reply:
Oh, yes, you're right. Fixed.